presto-product-tests/src/main/java/com/facebook/presto/tests/SystemConnectorTests.java

```diff
@@ -55,10 +57,18 @@ public void selectRuntimeQueries()
                 "  last_heartbeat," +
                 "  'end' " +
                 "FROM system.runtime.queries";
-        assertThat(query(sql))
-                .hasColumns(VARCHAR, VARCHAR, VARCHAR, VARCHAR, VARCHAR, ARRAY,
-                        BIGINT, BIGINT, BIGINT, TIMESTAMP, TIMESTAMP, TIMESTAMP, VARCHAR)
-                .hasAnyRows();
+        if (usingTeradataJdbcDriver(defaultQueryExecutor().getConnection())) {
```

Comment:
It's hard to see the difference between the two assertions. What if we restructure as
```java
JDBCType arrayType = usingTeradataJdbcDriver(defaultQueryExecutor().getConnection()) ? VARCHAR : ARRAY;
assertThat(query(sql))
        .hasColumns(VARCHAR, VARCHAR, VARCHAR, VARCHAR, VARCHAR, arrayType,
                BIGINT, BIGINT, BIGINT, TIMESTAMP, TIMESTAMP, TIMESTAMP, VARCHAR)
        .hasAnyRows();